hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenFeeScheduleUpdateHandler.java

```diff
@@ -126,8 +127,7 @@ private Token validateSemantics(
             @NonNull final TokenFeeScheduleUpdateTransactionBody op,
             @NonNull final WritableTokenStore tokenStore,
             @NonNull final TokensConfig config) {
-        var token = tokenStore.get(op.tokenIdOrElse(TokenID.DEFAULT));
-        validateTrue(token != null, INVALID_TOKEN_ID);
+        var token = TokenHandlerHelper.getIfUsable(op.tokenIdOrElse(TokenID.DEFAULT), tokenStore);
```

Comment:
I see three different approaches being used here:

```var token = TokenHandlerHelper.getIfUsable(op.tokenIdOrElse(TokenID.DEFAULT), tokenStore);```

```
final var token = tokenStore.get(tokenId);
validateFalse(token != null && token.deleted(), TOKEN_WAS_DELETED);
```

```
final var tokenId = context.body().tokenMintOrThrow().tokenOrThrow();
final var token = TokenHandlerHelper.getIfUsable(tokenId, tokenStore);
```

Is there a common way we should be doing this, or are each of these correct? I didn't dig into it further, it just struck me as odd that checking the token ID is being done differerently

Reply:
You're right, i've changed some and tried to make them similar. I agree that we should be consistent.

Reply:
Generally we want to use `getIfUsable(...)` because it checks for nonexistent, deleted, or paused tokens. However in some cases that's not appropriate, for example handling an unfreeze token call, so either we go back to checking a subset of those conditions manually, or we come up with various permutations of `getIfUsable(...)`. We've tried to only use `tokenStore.get(...)` in places where `getIfUsable` isn't correct, but I'm sure we've missed a spot or two